pynetsnmp/netsnmp.py

```diff
@@ -857,7 +874,7 @@ def sendTrap(self, trapoid, varbinds=None):
                 n = strToOid(n)
                 lib.snmp_add_var(pdu, n, len(n), t, v)
 
-        lib.snmp_send(self.sess, pdu)
+        lib._snmp_send(self.sess, pdu)
```

Comment:
should these not be self._send_snmp instead of lib._send_snmp?

Reply:
yes,I've added changes